static-assets/components/cstudio-browse/browseCMIS.js

```diff
@@ -282,6 +282,24 @@
                 }
             }
             CStudioAuthoring.Service.contentCloneCMIS(paramsJson, callbackContent);
+
+            const dialogContent = `<div class="loading-bar">
```

Comment:
Rubbish thing about using template quotes is that the string ends up with all that tab space on subsequent lines... Alternatively you could
- do array and join at the end 
- simple string concat
- consts/function declared at the top of the file with minimal indentation - In this case there's no interpolation so even simple consts would do.

To keep spaces on arrays, simply put spaces at the top of the string, or use block comments to create indentation.

Reply:
done, updated to string concat